lib/smart_city/event/dataset_update/_metadata.ex

```diff
@@ -0,0 +1,60 @@
+defmodule SmartCity.Event.DatasetUpdate.Metadata do
```

Comment:
Do we have any apps that use metadata?
If not, it'd be nice to see this go away (though I wouldn't want that to happen in this card/pr).

Reply:
The dataset update event struct uses metadata as a field

Reply:
I think forklift uses it to store kafka keys and timing data